.github/workflows/precommit.yml

```diff
@@ -246,15 +246,13 @@ jobs:
             .tox/coverage.xml
 
   test-enable-cte-optimization:
-    name: Test Enable CTE Optimization py-${{ matrix.os.download_name }}-${{ matrix.python-version }}-${{ matrix.cloud-provider }}
+    name: Test Enable CTE Optimization py-${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.cloud-provider }}
```

Comment:
```suggestion
    name: Test Enable CTE Optimization (only doctests) py-${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.cloud-provider }}
```

Reply:
actually not only doctest, but exclude doctest